core/shared/src/main/scala-2.11/zio/NotExtends.scala

```diff
@@ -16,13 +16,13 @@
 
 package zio
 
-import zio.clock.Clock
-import zio.console.Console
-import zio.system.System
-import zio.random.Random
-import zio.internal.{ Platform, PlatformLive }
+@annotation.implicitNotFound(
+  "The environment ${A} already contains service ${B}, are you sure you want to overwrite it? Use Has#update to update a service already inside the environment."
```

Comment:
The implicit will always be found, so it's safe to delete this.

Reply:
If the implicit is ambiguous on Scala 2.11 that will be treated as the implicit not being found and trigger the message. Are you sure you want to delete this?

Reply:
Ah, I didn't know that! Let's keep it then. 👍